### ISSUE TITLE
edit_entry help requested

### ISSUE THREAD

**Issue by jberanek** [meeting-room-booking-system/mrbs-code#2010]
[Transferred from the general mailing list]

We currently successfully run MRBS 1.4.8 for our local association (culture and sports).
What happens is that, even when there are days off like vacations, we may need to book rooms on an occasional basis for special meetings.
At the time of 1.4.8, I could find out how to add an additional checkbox and the necessary code to allow this overwrite operation so that it looks like ‘closed’ and yet allow for adding these special meetings.
Since then, MRBS has fortunately evolved a lot and looks now much better but the drawback is that the code has changed so much that I’ve been unable so far to figure out where to do this change.

If anyone could  point me in the right direction, this would be greatly appreciated.
Basically I just need to be able to add a booking no matter what the room status is.
Since there is only one person creating bookings, I will be adding a checkbox to the edit_entry screen to allow her to ‘overwrite’ the closed status of the room on demand and create the requested booking.

Reported by: campbell-m

Original Ticket: [mrbs/support-requests/1294](https://sourceforge.net/p/mrbs/support-requests/1294)
